app/src/components/login/Login.jsx

```diff
@@ -10,7 +11,7 @@ export default function Login() {
 
   const handleLogin = (e) => {
     e.preventDefault();
-    Login({ email, password }, dispatch);
+    login({ email, password }, dispatch);
```

Comment:
The apiCall login function was not imported, it was using the Login component which was causing errors.